lib/mayaUsd/nodes/proxyShapeBase.h

```diff
@@ -211,6 +218,14 @@ class MayaUsdProxyShapeBase : public MPxSurfaceShape,
                 const MEvaluationNode& evaluationNode,
                 PostEvaluationType evalType) override;
 
+#if MAYA_API_VERSION >= 20210000
```

Comment:
`configCache` and `getCacheSetup` are also available in 2020. why are we allowing only 2021 here?

Reply:
Exactly, ProxyAccessor is only available in PreviewRelease because it depends on a number of fixes done on the Maya side. For this particular case, you are correct that cache interfaces are available in Maya 2020, but nothing will use them and we would still need to add conditional compilation for older versions of Maya. This is why I decided to keep it simple and turn it all off for released versions of Maya.

Reply:
understood! thanks!